src/app/custom-global-scss/custom-global-scss.component.html

```diff
@@ -0,0 +1,38 @@
+<aw-wizard class="custom-global-scss">
+  <aw-wizard-step [stepTitle]="'Steptitle 1'">
+    <div class="centered-content">
+      <div>
+        <h3>Custom CSS</h3>
```

Comment:
The `h3` text is wrong. The example does not show "Custom CSS" but "Custom SCSS". In addition it is not clear how this example differs from the `custom css` one (both examples seem to show the same content?). I would also like some emphasis on how the changed color palete is actually applied inside the custom css/scss, i.e. on:

```scss
  // Generate styles for our custom wizard
  @include aw-define-styles();
```

Reply:
Added an elaborate description in 79c4081a0fdd35269c144040fad6754232129e07